suclass/cms/templates/body-post.html

```diff
@@ -1 +1,12 @@
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85"></script>
+<script>
+	$(document).ready(
+		setTimeout(
+			function()
+			{
+				var s = document.createElement("script");
+				s.id = "ze-snippet"
+				s.src = "https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85";
+				$("head").append(s)
+			}, 1000)
+	);
+</script>
```

Comment:
Which text editor are you using?

See the red icons here? It looks like yours isn't properly configured and as such, is _not_ appending a trailing newline to files (which is the POSIX standard for text files).

If you're using `vi`, you can add this to your `~/.vimrc`:

```
set endofline "write <EOL> for last line in file
```

If you're using another editor, you'll need to poke around its settings.

Reply:
@aamirrasheed I'm still seeing this one here; let me know if you need a hand.